components/camera/camera.go

```diff
@@ -41,8 +41,18 @@ func init() {
 	}, newReadImageCollector)
 }
 
-// SubtypeName is a constant that identifies the camera resource subtype string.
-const SubtypeName = "camera"
+const (
+	// SubtypeName is a constant that identifies the camera resource subtype string.
+	SubtypeName = "camera"
+
+	// TimeRequestedMetadataKey is optional metadata in the gRPC response header that correlates
```

Comment:
I actually like having this in `camera` so it can be used for other camera implementations if needed. I think as long as it's well-documented, it's okay. Thoughts @edaniels ?

Reply:
Okay, done!